### PR TITLE
improve design of formatdialog

### LINKED ISSUE #283: setup: fix format dialog

### PR DESCRIPTION
**Pull request by cstenglein**
fixes #283

### CODE REVIEW

File: src/components/Setup/FormatDialog.tsx [hunk 1]
```diff
@@ -28,29 +29,34 @@ const FormatDialog: FC<Props> = ({ containsBlockchain, callback }) => {
...
-            className="bd-button my-5 p-2"
-          >
-            {t("setup.blockchain_keep")}
-          </button>
-        </div>
+        <section className="flex h-full flex-col items-center justify-center p-8">
+          <h2 className="m-2 text-center text-lg font-bold">
+            {t("setup.blockchain_found_short")}
+          </h2>
+          <div className="text-sm">{t("setup.blockchain_found_long")}</div>
```
Comment (escapedcat) [inline src/components/Setup/FormatDialog.tsx:36]: nit: Could be `p`?

File: src/components/Setup/FormatDialog.tsx [hunk 2]
```diff
@@ -28,29 +29,34 @@ const FormatDialog: FC<Props> = ({ containsBlockchain, callback }) => {
...
+          <article className="mt-10 flex flex-col items-center justify-center gap-10 md:flex-row">
+            <button
```
Comment (escapedcat) [inline src/components/Setup/FormatDialog.tsx:38]: nit: There' s no form around these buttons, right? Could those be `a`?

File: src/components/Setup/FormatDialog.tsx (hunk 1 above)
Comment (cstenglein) [reply src/components/Setup/FormatDialog.tsx:36]: Yes, thank you! :)

File: src/components/Setup/FormatDialog.tsx (hunk 2 above)
Comment (cstenglein) [reply src/components/Setup/FormatDialog.tsx:38]: Yeah, but it's still a button and it doesn't thange the URL of the page, just changes the component,, so I don't think a `a` would be fitting here

**Pull request merged**